.github/workflows/pr-title-commitlint-check.yml

```
@@ -0,0 +1,16 @@
+name: PR Title Commitlint Check
+
+on:
+  pull_request:
+    types: [opened, synchronize, edited]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+      - name: Install dependencies
+        run: npm install
```

Clean install is usually a bit faster:

```suggestion
        run: npm ci
```